debian/gen-empty-fixtures.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -eux
+
+SRCDIR=$(readlink -f "$(dirname "$0")")
+TMPDIR=$(mktemp -d)
+OUTPUTDIR=$(realpath "${SRCDIR}/../$1")
+
+trap 'rm -rf "${TMPDIR}"' EXIT
+
+cd "${TMPDIR}"
+
+echo 'Suite: mythology
+Codename: ragnarok
+Architectures: ppc64
+Components: asgard' > Release
+
+apt-ftparchive release . >> Release
+
+mkdir -p "${OUTPUTDIR}"/dists/fjalar/
+cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/Release -t "${OUTPUTDIR}"/dists/fjalar/
```

Comment:
This looks like a hell of a command to copy a single file. Am I missing something?
Also what about the output if apt-ftparchive here?
If the "empty repository" is just a release file, we can short cut this a lot i think.
Also should the name of the dists directory not match the codename?

Reply:
the cp command is just copy paste from the other scripts so for consistency sake

it's an empty repository without any packages iirc but it should include a release file